GDevelop.js/__tests__/Core.js

```diff
@@ -4295,19 +4295,37 @@ describe('libGD.js', function () {
       expect(objectsContainer.hasObjectNamed('MyObjectWithoutType')).toBe(true);
       expect(objectsContainer.hasObjectNamed('MySpriteObject')).toBe(true);
 
-      const objectWithoutType = objectsContainer.getObject('MyObjectWithoutType');
-      expect(objectWithoutType.getType()).toBe(
-        ''
+      const objectWithoutType = objectsContainer.getObject(
+        'MyObjectWithoutType'
       );
+      expect(objectWithoutType.getType()).toBe('');
       const mySpriteObject = objectsContainer.getObject('MySpriteObject');
       expect(objectsContainer.getObject('MySpriteObject').getType()).toBe(
         'Sprite'
       );
 
-      // Check that behaviors were also added.
-      expect(objectsContainer.getObject('MySpriteObject').getAllBehaviorNames().toJSArray()).toEqual(
-        ['MyFirstBehavior', 'MySecondBehavior']
-      );
+      // Check that behaviors were also added AND that default behaviors are also present.
```

Comment:
Maybe `ObjectMetadata::AddDefaultBehavior` can be used to simulate a change in a custom object.